### PR TITLE
Standardise file names - Closes #312

### LINKED ISSUE #312: Rename camelCase files
Child of #426 

For improved compatibility.

### PR DESCRIPTION
**Pull request by shuse2**
Closes #312

### CODE REVIEW

File: src/commands/create_account.js [hunk 1]
```diff
@@ -13,7 +13,7 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-import cryptoModule from '../utils/cryptoModule';
+import cryptoModule from '../utils/crypto_module';
```
Comment (willclarktech) [inline src/commands/create_account.js]: How about renaming this file `cryptography` while we're here?

Comment (shuse2) [reply src/commands/create_account.js]: I think it would be great, but should we do it in this PR?

Comment (willclarktech) [reply src/commands/create_account.js]: This issue is about standardizing file names, I think it's covered.

Comment (shuse2) [reply src/commands/create_account.js]: ok ill change the PR title too

**Pull request merged**